metricsx/middleware.go

```diff
@@ -288,6 +292,45 @@ func (sw *Service) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.
 	}
 }
 
+func (sw *Service) UnaryInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+	var start time.Time
+	if !sw.optOut {
+		start = time.Now()
+	}
+
+	resp, err := handler(ctx, req)
+
+	if sw.optOut {
+		return resp, err
+	}
+
+	latency := time.Since(start) / time.Millisecond
+
+	if err := sw.c.Enqueue(analytics.Page{
```

Comment:
Can `info` contain any sensitive information like UUIDs or similar?

Reply:
It only has two fields:
```go

// UnaryServerInfo consists of various information about a unary RPC on
// server side. All per-rpc information may be mutated by the interceptor.
type UnaryServerInfo struct {
	// Server is the service implementation the user provides. This is read-only.
	Server interface{}
	// FullMethod is the full RPC method string, i.e., /package.service/method.
	FullMethod string
}
```
which are not sensitive as far as I can tell. The `FullMethod` does not contain parameters.